apps/ensindexer/src/plugins/multi-chain/Resolver.ts

```diff
@@ -20,15 +20,14 @@ import {
 /**
  * Shared Resolver indexing functions should be registered exactly once, or Ponder will complain about
  * multiple indexing functions being registered for these events. This boolean allows this function
- * to be idempotent — many plugins can call it, but only one will succeed,
- * which is enough to correctly register multi-network Resolver indexing handlers.
+ * to be idempotent — many plugins can call it, but only the first will take effect.
  */
 let hasBeenRegistered = false;
 
 /**
  * Registers event handlers with Ponder.
  */
-export default function attach_SharedResolverHandlers() {
+export default function attach_SharedMultichainResolverHandlers() {
```

Comment:
Wondering if this file should be moved to `apps/ensindexer/src/handlers/` directory, so the plugins directory can only include plugin directories?